src/components/MessageList/MessageList.tsx

```diff
@@ -250,6 +257,8 @@ export const MessageList = <
     setFlatListRef,
     threadList,
     TypingIndicator = DefaultTypingIndicator,
+    NewMessageNotification = MessageNotification,
```

Comment:
```suggestion
    MessageNotification = MessageNotificationDefault,
```

Reply:
also put this and the onListScroll in alphabetical order like the rest of the props here